core/ui/vapor_ui/qam/powertools_menu.gd

```diff
@@ -74,6 +75,23 @@ func _ready():
 		gpu_temp_slider.value_changed.connect(_on_gpu_temp_limit_changed)
 		gpu_temp_slider.visible = gpu.tj_temp_capable
 
+	if gpu.thermal_mode_capable:
+		thermal_profile_dropdown.clear()
+		thermal_profile_dropdown.add_item("Balanced", 0)
+		thermal_profile_dropdown.add_item("Performance", 1)
+		thermal_profile_dropdown.add_item("Silent", 2)
+		thermal_profile_dropdown.visible = gpu.thermal_mode_capable
+		var thermal_mode := int(_read_sys("/sys/devices/platform/asus-nb-wmi/throttle_thermal_policy"))
```

Comment:
This seems very ASUS platform-specific. We probably want to keep this path in the Ally platform itself instead of hard-coding it in the powertools menu.